main.js

```diff
@@ -0,0 +1,101 @@
+// --- DOM Elements --- \\
+const valuePlayer_1 = document.getElementById("player1-score--value");
+const valuePlayer_2 = document.getElementById("player2-score--value");
```

Comment:
why are we using snake case? What is this Python? Use `camelCase` for variable names please

Reply:
Sry looked at these again. Maybe consistency is the call out here